ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/FinalProjectNode.java

```diff
@@ -142,16 +165,6 @@ public void validateKeyPresent(final SourceName sinkName) {
   private void validate() {
```

Comment:
nit: it's a bit odd that this method is called `validate` now that there's validation being performed in `build()` as well. Can we either move the validation back into this method, or rename this method to something more specific (`throwOnZeroValueColumns()` or similar)?